j2k/testData/fileOrElement/strings/formattedString.kt

```diff
@@ -4,7 +4,7 @@ object A {
             "text3"
 
 
-    val TEXT2 = "text1\n"
-    + "text2\n"
-    + "text3"
+    val TEXT2 = ("text1\n"
```

Comment:
If I were you, I would reduce space from 2 to 1 line. I think one is enough to make this object readable.